lambdas/pkgselect/index.py

```diff
@@ -72,47 +72,13 @@ def get_s3_client(
     return s3_client
 
 
-def call_s3_select(
-    s3_client: str,
-    bucket: str,
-    key: str,
-    logical_key_prefix: str,
-    detail=False
-):
+def fix_quotes(key: str):
```

Comment:
Dima already has something for this in access_counts, just factor that out into shared as well:
https://github.com/quiltdata/quilt/blob/master/lambdas/access_counts/index.py#L34